colour/characterisation/tests/test_cameras.py

```diff
@@ -26,31 +24,7 @@ class TestRGB_SpectralSensitivities(unittest.TestCase):
     class units tests methods.
     """
 
-    def test_required_attributes(self):
-        """
-        Tests presence of required attributes.
-        """
-
-        required_attributes = ('name', 'mapping', 'labels', 'data', 'x', 'y',
-                               'z', 'wavelengths', 'values', 'shape', 'red',
-                               'green', 'blue')
-
-        for attribute in required_attributes:
-            self.assertIn(attribute, dir(RGB_SpectralSensitivities))
-
-    def test_required_methods(self):
-        """
-        Tests presence of required methods.
-        """
-
-        required_methods = ('__hash__', '__getitem__', '__setitem__',
-                            '__iter__', '__contains__', '__len__', '__eq__',
-                            '__ne__', '__add__', '__sub__', '__mul__',
-                            '__div__', 'get', 'extrapolate', 'interpolate',
-                            'align', 'zeros', 'normalise', 'clone')
-
-        for method in required_methods:
-            self.assertIn(method, dir(RGB_SpectralSensitivities))
+    pass
```

Comment:
Is there a specific reason for this empty test class?

Reply:
I was holding those tests in case I would need to update them further down the line, I think we should remove them now.